packages/components/tests/e2e/specs/vsCodeExtension.spec.js

```diff
@@ -747,6 +747,35 @@ context('VS Code Extension', () => {
       cy.get('.trace-node[data-event-id="15"]')
         .should('be.visible')
         .should('have.class', 'highlight');
+
+      cy.get('.trace-node[data-event-id="13"]')
+        .click()
+        .should('have.class', 'highlight');
+      cy.get('.trace-node[data-event-id="1"]').should(
+        'not.have.class',
```

Comment:
The filtered nodes should retain the `filtered` class despite the active selection not being part of the filtered set.

Reply:
fixed https://github.com/applandinc/appmap-js/pull/519/commits/c9bd5fee3eba5272ed597d26dd4775d7a8294dcc